docs/usage/create_dataset.rst

```diff
@@ -71,14 +71,15 @@ You can generate your own Epitome dataset from ENCODE using the following comman
                           http://hgdownload.cse.ucsc.edu/admin/exe/
 
 
-To use your own dataset in an Epitome model, make sure to set the environment environment variable
-``EPITOME_DATA_PATH`` that points to your custom dataset. This will tell Epitome where to load
-data from.
+To use your own dataset in an Epitome model, make sure to specify the ``data_dir``
+and/or ``assembly`` variables when creating the ``EpitomeDataset`` class. This
+will tell Epitome where to load data from.
```

Comment:
and we comment on what happens if only one is specified, or both? Or we can link to the section that explains this

Reply:
Added comments for both